doc/api/stream.md

```diff
@@ -2517,6 +2517,28 @@ run().catch(console.error);
 after the `callback` has been invoked. In the case of reuse of streams after
 failure, this can cause event listener leaks and swallowed errors.
 
+`stream.pipeline()` closes all the streams when an error is raised.
+The `IncomingRequest` usage with `pipeline` could lead to an unexpected behavior
```

Comment:
What is `IncomingRequest`? Did you mean `IncomingMessage`? In the example below `res` is a `ServerResponse` or `OutgoingMessage`.

Reply:
Actually, I meant an `Incoming Request` in general. The idea is to show that a socket/request usage with pipeline could lead to a error